components/browser/icons/src/main/java/mozilla/components/browser/icons/BrowserIcons.kt

```diff
@@ -110,9 +120,7 @@ class BrowserIcons(
             is IconLoader.Result.BitmapResult -> Icon(result.bitmap, source = result.source)
 
             is IconLoader.Result.BytesResult ->
-                decode(result.bytes, decoders, desiredSize)?.let { bitmap ->
-                    Icon(bitmap, source = result.source)
-                } ?: return generator.generate(context, request)
+                decode(result.bytes, decoders, desiredSize)?.let { Icon(it, source = result.source) }
```

Comment:
I do not understand how this change is related and why we make it: Why would we want to continue to run through the processors with a `null` icon?

Reply:
I think I made this change because the linter complained at one point. `process` returns immediately if the icon is null so the effect is the same. I'll remove these changes.